### PR TITLE
Usage example to upload to data to FDP

### PR DESCRIPTION
**Pull request by a-nayden**
Usage example to upload to data to FDP

### CODE REVIEW

File: docs/Usage_example_FDP.ipynb [hunk 1]
```diff
@@ -0,0 +1,558 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8bff32b5-6f5c-4c6b-953b-e8f9a8ddd681",
+   "metadata": {},
+   "source": [
+    "**Prerequirements:** To execute this notebook in full one needs to have a running Fair Data Point (FDP) instance with an active write access account.\n",
+    "\n",
+    "Let us consider uploading datasets from .example_data.csv to FDP. \n",
```
Comment (kburger) [inline docs/Usage_example_FDP.ipynb:18]: Leading `.` before the example file name, but maybe that is intentional?
```suggestion
    "Let us consider uploading datasets from example_data.csv to FDP. \n",
```

File: docs/Usage_example_FDP.ipynb [hunk 2]
```diff
@@ -0,0 +1,558 @@
...
+   "source": [
+    "# Preparing and uploading data to Fair Data Point with SeMPyRO"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8bff32b5-6f5c-4c6b-953b-e8f9a8ddd681",
+   "metadata": {},
+   "source": [
+    "**Prerequirements:** To execute this notebook in full one needs to have a running Fair Data Point (FDP) instance with an active write access account.\n",
```
Comment (kburger) [inline docs/Usage_example_FDP.ipynb:16]: Corrected spelling.

```suggestion
    "**Prerequirements:** To execute this notebook in full one needs to have a running FAIR Data Point (FDP) instance with an active write access account.\n",
```

File: docs/Usage_example_FDP.ipynb [hunk 3]
```diff
@@ -0,0 +1,558 @@
...
+    "catalog_record = catalog.to_graph(catalog_subject)\n",
+    "print(catalog_record.serialize())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "515557de-b2e3-4fb3-a25f-f92b291f1b08",
+   "metadata": {},
+   "source": [
+    "Now we need to define an FDP API client which can log into an FDP, get a token and perform basic calls.\n"
```
Comment (kburger) [inline docs/Usage_example_FDP.ipynb:66]: In an ideal world there's a package for the fdp client, but this probably does not exist yet? Maybe something to consider for Mark's work as well.

**Review by kburger — COMMENTED** [review]
Minor comments added.

File: docs/Usage_example_FDP.ipynb (hunk 3 above)
Comment (Markus92) [reply docs/Usage_example_FDP.ipynb:66]: Agreed on that an FDP client would be desirable.
There is an FDP client from NLeSC which is very similar to what we have, but it's unmaintained (see https://github.com/fair-data/fairdatapoint-client). There's no authentication support, either.

We could consider splitting off our own FDP client into a separate package?

Comment (a-nayden) [reply docs/Usage_example_FDP.ipynb:66]: imho we should

**Review by Markus92 — APPROVED** [review]
lgtm

**Pull request merged**